File: two-pointer/js/reverse-array.js
const arr = [10, 20, 30, 40, 50, 100];

function reverseArrayTwoPointerAlog(arr) {
  start = 0;
  end = arr.length - 1;

  while (start < end) {
    let temp = arr[start];
    arr[start] = arr[end];
    arr[end] = temp;
    start++;
    end--;
  }

  return arr;
}

console.log(reverseArrayTwoPointerAlog(arr));

File: two-pointer/python/reverse-array.py

arr = [10, 20, 30, 40, 50, 100]

def reverse_array_two_pointer_algo(arr):
 start = 0
 end = len(arr)-1

 while start <= end:
   arr[start] ,arr[end] = arr[end],arr[start]
   start += 1  
   end -= 1

 return arr


print(reverse_array_two_pointer_algo(arr))